src/components/app-switcher/README.md

```diff
@@ -0,0 +1,17 @@
+# App Wrapper
+
+## How to use a AppSwitcher in a component:
+
+* In your file
+
+```javascript
+import AppWrapper from 'carbon/lib/components/app-switcher';
+```
+
+*  To render a AppWrapper:
+
+```javascript
+  <AppSwitcher manuTitle="Applications" applicationJson='{"items":[{"title":"CLIENT APPS","items":[{"name":"Accounting","href":"#accounting"},{"name":"Payroll","href":"#payroll"}]},{"title":"ACCOUNTANT APPS","items":[{"name":"Accountants Cloud","href":"#accountantCloud"},{"name":"Corporation Tax","href":"#corporateTax"},{"name":"Final Accounts","href":"#finalAccounts"}]}]}'>
```

Comment:
Yeah, our requirements for this component were rigid, but with a desire to push it to carbon. I'll make it more generic, maybe more like a generic drawer